sideboard/lib/_websockets.py

```diff
@@ -490,4 +509,95 @@ def _callback(self, response_data):
 
     def callback(self, response_data):
         """override this to define what to do with your rpc method return values"""
-        pass
+
+
+class MultiSubscription(object):
```

Comment:
MAGFest doesn't care about this, though one day it might.  I think the docstring should explain everything.